OAuth2.xcodeproj/project.pbxproj

```diff
@@ -572,7 +580,7 @@
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "Mac Developer: Renaud Boisjoly (XB4BFU6P3B)";
```

Comment:
Can you set this to the generic `Mac Developer` instead of your specific profile?